lib/govuk_app_config/govuk_error.rb

```diff
@@ -1,4 +1,5 @@
 require "sentry-raven"
+require "govuk_app_config/govuk_error/configuration"
```

Comment:
Note to self: recode/rebase so that the delegator pattern comes _first_

_then_ the code to add hardcoded data sync window inside the should_capture

_then_ the code to generalise that in an ENV.